packages/itwinui-css/src/searchbox/searchbox.scss

```diff
@@ -12,10 +12,15 @@
   }
 }
 
-// noop https://github.com/iTwin/iTwinUI/pull/1881#discussion_r1506166537
-/* stylelint-disable-next-line block-no-empty */
 .iui-search-icon {
+  --_iui-button-icon-fill: var(--iui-color-icon-muted);
```

Comment:
Ok, this is a weird one, let me explain…after https://github.com/iTwin/iTwinUI/pull/1919/commits/c07bf5572e7e0f2a28a8eb37806dc78ce1c29817 the search input's search icon started failing CSS test images.

![image](https://github.com/iTwin/iTwinUI/assets/849817/6d5547cf-5de7-47b1-ada5-af6ff52d1902)

I found this odd because that commit didn't change any CSS, so I thought previous commits would have failed test images.

The reason it's failing has to do with a specificity or CSS order within `all.scss`.

| localhost | production |
| --- | --- |
| ![image](https://github.com/iTwin/iTwinUI/assets/849817/d7d039d5-460b-4251-a3ec-bb621730eccb) | ![image](https://github.com/iTwin/iTwinUI/assets/849817/98836f33-cbaf-4369-b935-51262b69436a) |

One solution was to undo the noop of the `.iui-search-icon` class.  Another option is to undo the re-order of `utils/utils` within `all.scss`, but that caused specificity issues with `iui-field`.

Reply:
See https://github.com/iTwin/iTwinUI/pull/1919#discussion_r1576805293

Reply:
All reverted and working now. https://github.com/iTwin/iTwinUI/pull/1919#discussion_r1578231027